util/mknum.pl

```diff
@@ -138,8 +138,9 @@
     }
     if ($dropped) {
         print STDERR "${ordinals_file}: Dropped $dropped new symbols\n";
+        $stats{unassigned} = 0 unless defined $stats{unassigned};
+        $unassigned = $stats{unassigned} - $dropped;
```

Comment:
Shouldn't those 2 lines be outside the if() block?

Reply:
> Shouldn't those 2 lines be outside the if() block?

`$stats{unassigned}` isn't used except here, and why subtract zero? :)  So no.

Reply:
Huh? But `$unassigned` is used down below and you're happily leaving it without any value assigned.

Reply:
> Huh? But `$unassigned` is used down below and you're happily leaving it without any value assigned.

replacing one 'unassigned' bug with another. :)  fixed, thanks. udpated commit pushed.